### PR TITLE
Hide active mask details from ert_run_context implementation

### PR DESCRIPTION
**Pull request by joakim-hove**
**Task**
Some changes in the `ert_run_context` implementation - less leakage of internal details.

**Approach**
_Short description of the approach_

**Pre un-WIP checklist**
- [ ] Statoil tests pass locally
- [x] Have completed graphical integration test steps

**Depends on**
Statoil/ert#85

### CODE REVIEW

File: libenkf/include/ert/enkf/ert_run_context.h [hunk 1]
```diff
@@ -46,14 +46,14 @@ typedef struct ert_run_context_struct ert_run_context_type;
                                                enkf_fs_type * target_update_fs ,
                                                bool_vector_type * iactive ,
                                                path_fmt_type * runpath_fmt ,
-					       const char * jobname_fmt,
+                 const char * jobname_fmt,
```
Comment (andreabrambilla) [inline libenkf/include/ert/enkf/ert_run_context.h]: If I were the maintainer I'd hate you :smile:

File: libenkf/include/ert/enkf/ert_run_context.h [hunk 2]
```diff
@@ -67,7 +67,7 @@ typedef struct ert_run_context_struct ert_run_context_type;
   ert_run_context_type * ert_run_context_alloc_SMOOTHER_RUN(enkf_fs_type * sim_fs , enkf_fs_type * target_update_fs ,
                                                             bool_vector_type * iactive ,
                                                             const path_fmt_type * runpath_fmt ,
-							    const char * jobname_fmt,
+                  const char * jobname_fmt,
```
Comment (andreabrambilla) [inline libenkf/include/ert/enkf/ert_run_context.h]: indentation

File: libenkf/src/enkf_main.c [hunk 3]
```diff
@@ -1343,7 +1343,7 @@ void * enkf_main_icreate_run_path( enkf_main_type * enkf_main, run_arg_type * ru
                       run_arg_get_iens( run_arg ),
                       run_arg_get_iter( run_arg ),
                       run_arg_get_runpath( run_arg ),
-		      run_arg_get_job_name( run_arg ));
+          run_arg_get_job_name( run_arg ));
```
Comment (andreabrambilla) [inline libenkf/src/enkf_main.c]: indentation

File: libenkf/src/enkf_main.c [hunk 4]
```diff
@@ -2305,16 +2303,15 @@ int enkf_main_load_from_forward_model_with_fs(enkf_main_type * enkf_main, int it
...
+                                                                                  model_config_get_runpath_fmt( model_config ),
+                                                                                  model_config_get_jobname_fmt( model_config ),
+                                                                                  enkf_main_get_data_kw(enkf_main),
+                                                                                  iter );
   arg_pack_type ** arg_list = util_calloc( ens_size , sizeof * arg_list );
   thread_pool_type * tp     = thread_pool_alloc( 4 , true );  /* num_cpu - HARD coded. */
 
-  int iens = 0;
-  for (; iens < ens_size; ++iens) {
+  for (int iens; iens < ens_size; ++iens) {
```
Comment (andreabrambilla) [inline libenkf/src/enkf_main.c]: iens not initialized here

File: libenkf/include/ert/enkf/ert_run_context.h (hunk 1 above)
Comment (joakim-hove) [reply libenkf/include/ert/enkf/ert_run_context.h]: yes - I can see that; I had some editor f..up with tabs and space, tried to fix it, but some obviously fell through - will fix it.

**Review by andreabrambilla — CHANGES_REQUESTED** [review]
I'm always happy to see parameters disappearing from a method's signature :+1:

Just fix the uninitialized `iens`, then this is good to be merged IMHO

**Comment by joakim-hove** [comment]
Thank you for the review - changes pushed.

**Pull request merged**